html/index.html

```diff
@@ -3,9 +3,213 @@
 <head>
   <meta charset="UTF-8">
   <title>Document</title>
+  <link href='css/less/styles.css' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css?family=Raleway:700|Montserrat&display=swap" rel="stylesheet">
 </head>
 <body>
-  
-    <script src="http://localhost:35729/livereload.js"></script>
+  <nav>
+    <a href='#' id='home_link'>Adam Dow</a>
+    <a href='#'>Improv</a>
+    <a href='#'>Services</a>
+    <a href='#'>Gallery</a>
+    <a href='#'>About</a>
+    <a href='#'>Contact</a>
+  </nav>
+  <div>
+    <img src='#' alt='#' />
+    <div>
+      <h3>Improv Shows</h3>
+      <p>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellendus hic natus voluptas nobis itaque fugit
+        aliquid voluptate eveniet! In dolores, sed rerum quam velit ipsa culpa illum aut maiores facilis.
+      </p>
+      <button>Read More</button>
```

Comment:
all `<button>` should be a `<a>` because it will link to a section on the same page.